addons/reload/ACE_Arsenal_Stats.hpp

```diff
@@ -0,0 +1,13 @@
+class EGVAR(arsenal,stats) {
+    class statBase;
+    class ACE_isBelt: statBase {
+        scope = 2;
+        priority = -1;
+        stats[] = {"ACE_isBelt"};
+        displayName = CSTRING(LinkBelt);
+        showText = 1;
+        textStatement = QUOTE(localize QUOTE(ELSTRING(Common,Enabled)));
```

Comment:
```suggestion
        textStatement = ECSTRING(Common,Enabled);
```

Will this work?

Reply:
it will look like 
`call compile "ace_common_x"`
and return nil, which is why I got the "800 rpm" weirdness

Reply:
arsenal action does 
```
        private _text = getText (_x >> "text");
        private _textStatement = getText (_x >> "textStatement");

        private _type = switch (false) do {
            case (_text == ""): {
                _statement = format ["""%1""", _text];
                ACTION_TYPE_TEXT
            };
            case (_textStatement == ""): {
                _statement = _textStatement;
                ACTION_TYPE_TEXT
            };
            case (_statement == ""): {
                _statement = _statement;
                ACTION_TYPE_BUTTON
            };
            default {
                -1
            };
        };
```
which sorta makes both compatible, but I think johnb432's code is fine